public/managehashes.php

```diff
@@ -47,7 +48,7 @@ function UpdateHashDetails(user, hash) {
             var date = new Date();
             var dateStr = date.getUTCDate() + ' ' + shortMonths[date.getUTCMonth()] + ' ' + date.getUTCFullYear() + ' ' + date.getUTCHours() + ':' + ('0' + date.getUTCMinutes()).slice(-2);
 
-            $('.comment-textarea').parents('tr').before('<tr class="comment system"><td></td><td class="w-full" colspan="3"><div><span class="smalldate">' + dateStr + '</span></div><div style="word-break: break-word">' + hash + ' updated by ' + user + '. Description: "' + name + '". Label: "' + labels + '"</div></td></tr>');
+            $('.comment-textarea').parents('tr').before('<tr class="comment system"><td></td><td class="w-full" colspan="3"><div><span class="smalldate">' + dateStr + '</span></div><div style="word-break: break-word">' + hash + ' updated by ' + user + '. File Name: "' + name + '". Label: "' + labels + '"</div></td></tr>');
```

Comment:
This is for the live update. You need to make a similar change in `updateHashDetails` for the value written to the database.

Reply:
Resolved in https://github.com/RetroAchievements/RAWeb/pull/1437/commits/df444e1c61a7289ea74e7c81ee4d0f0003d79af0